mobile/docs/root/development/releasing/releasing.rst

```diff
@@ -37,29 +37,31 @@ For example: January 25, 2020: ``0.3.1.20200125``.
 GPG Key
 ======================
 
-On 2024-04-20 the GPG key used to sign releases will expire. To extend the key's expiration date,
-follow these steps:
+NOTE: The `gpg` commands below may require running with `sudo`.
```

Comment:
not sure about this - if you run (some) with sudo then it will be roots keys you are working on

Reply:
i had to run it on my local machine to generate the keys.  those keys then get exported into GitHub secrets.  those secrets are used to do a gpg import of the keys on the CI machine so it can sign the AAR

Reply:
i guess other than not encouraging users to use root if its not necessary - my concern is that if you run some commands as sudo it adds a layer of confusion/mistakes if you then try to use gpg without

Reply:
why was it necessary to use root in this case?

Reply:
i actually retried this, and indeed i didn't need to `sudo` the command.  I believe I originally needed to do this while running in another terminal where I was getting operation cancelled, and this masked an issue where I needed to supply the right passphrase.  So I removed the note about needing `sudo` from the instructions.